melodic/distribution.yaml

```diff
@@ -630,6 +630,21 @@ repositories:
       url: https://github.com/ros/cmake_modules.git
       version: 0.4-devel
     status: maintained
+  cnn_bridge:
+    doc:
+      type: git
+      url: https://github.com/wew84/cnn_bridge.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/wew84/cnn_bridge.git
```

Comment:
This is also not a release repository. Note that if you use bloom then it will generate the PRs for you.